firstvoices/backend/tasks/import_job_tasks.py

```diff
@@ -250,36 +210,81 @@ def import_job_dry_run(data, import_job_instance):
 
 
 @shared_task
-def batch_import(import_job_instance_id, dry_run=True):
-    # This method passes the provided CSV file through the clean method,
-    # does a dry-run or the import as per the dry_run flag provided,
-    # then parses through the result to return a validation report.
-
+def batch_import_dry_run(import_job_instance_id):
```

Comment:
I like this as a separate method. Thumbs up. To keep with our usual way of using verbs to name functions, maybe this could be "validate_import_job"